### PR TITLE
shop.11freunde.de

### PR DESCRIPTION
**Pull request by gork7777**
issue: #81199

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -1221,7 +1221,7 @@ infobus.eu#?#body > #layer1:has(> div#layer2 > div.layer-message-container butto
 portaloswiatowy.pl#$#.overlay-cookie { display: none !important; }
 portaloswiatowy.pl#$#.overlay-hidden-cookie { overflow: visible !important; }
 easycosmetic.de##.footer-cookies
-easycosmetic.de##.overlay
+shop.11freunde.de,easycosmetic.de##.overlay
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_specific.txt:1224]: the site is broken with this rule, we need to accept cookies

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:1224]: Let me help you out @gork7777.

`11freunde.de#%#AG_onLoad(function() {if (!document.cookie.includes("euconsent-v2")) {var g = new MutationObserver(function() {var b = document.querySelector('#czgdpr-notice-wrp button[value="1"]');b&&b.click();});g.observe(document, {childList: !0,subtree: !0});setTimeout(function() {g.disconnect()}, 1E4)}});`

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/cookies_specific.txt:1224]: why euconsent-v2? 
`#czgdpr-notice-wrp button[value="1"]` this selector matches both buttons

I suggest:
11freunde.de#%#AG_onLoad(function() {if (!document.cookie.includes("gdpr_data")) {var g = new MutationObserver(function() {var b = document.querySelector('#czgdpr-notice-wrp #save');b&&b.click();});g.observe(document, {childList: !0,subtree: !0});setTimeout(function() {g.disconnect()}, 1E4)}});

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: I made mistake, in my User Rules, it's a different cookie name.

**Pull request merged**